src/com/epam/javatrainings/iohomework2/Customer.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.javatrainings.iohomework2;
+
+import javax.sql.XAConnection;
+
+public class Customer implements Comparable<Customer>{
+
+    private String name;
+    private String phoneNumber;
+    private int score;
+
+    public Customer(StringBuffer data){
```

Comment:
No need to use StringBuffer, it is too slow, you don't have a multithreading issue here. just simple String